pkg/rpcclient/actor/actor.go

```diff
@@ -25,6 +26,7 @@ import (
 // create and send transactions.
 type RPCActor interface {
 	invoker.RPCInvoke
+	SimpleWaiter
```

Comment:
BTW, the idea was to avoid adding it to the `RPCActor` (returning error from `Wait` if it can't be handled), but maybe we can live with it.

Reply:
Removed.

Reply:
You have `RPCPollingWaiter` there now.